commands/bootstrap_cli.py

```diff
@@ -190,6 +192,7 @@ def migrate_secrets(config_file, env, svc, overwrite, dry_run):
     AWS_PROFILE=myaccount copilot-bootstrap.py ...
     """
 
+    check_aws_conn(project_profile)
```

Comment:
I don't think we have this change covered by our tests. E.g. I commented it out and all the tests still passed.

Which reminds me, I keep meaning to look at adding mutation tests to our stuff.

Reply:
Agree with Will. Would be good to get a test in for this, if possible.

Reply:
@gabelton please add tests as per above prior to merge.

Reply:
Do you want a specific test for `aws_conn`? I had loads of failing tests when I made the changes, which I had to fix by adding various credentials logic and mocks to the migrate_secrets tests

Reply:
In the context of this pull request, I think a test to check that `make_config` exits when `check_aws_conn` exits will be enough to guard against us accidentally removing that behaviour.

Reply:
Added a test for `migrate_secrets` and started adding tests for utils, but will create a separate ticket to finish doing the latter